File: src/main/java/com/codecool/repvault/application/DTOs/incoming/ExerciseRequestDTO.java
package com.codecool.repvault.application.DTOs.incoming;

import com.codecool.repvault.domain.entities.enums.ExerciseEnum;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class ExerciseRequestDTO {
    private ExerciseEnum type;
    private String description;
    private List<SetRequestDTO> sets;
}

File: src/main/java/com/codecool/repvault/application/DTOs/incoming/ExerciseRequestDTO.kt
package com.codecool.repvault.application.DTOs.incoming

import com.codecool.repvault.domain.entities.enums.ExerciseEnum

data class ExerciseRequestDTO (
    val type: ExerciseEnum,
    val description: String,
    val sets: MutableList<SetRequestDTO>
)
